purchase_order_secondary_unit/models/purchase_order.py

```diff
@@ -18,8 +18,8 @@ class PurchaseOrderLine(models.Model):
 
     @api.depends("secondary_uom_qty", "secondary_uom_id")
     def _compute_product_qty(self):
-        if hasattr(super(), "_compute_product_qty"):
-            return super()._compute_product_qty()
+        self._compute_helper_target_field_qty()
+        return super()._compute_product_qty()
```

Comment:
Could you please clarify what the bug was?

Reply:
This is explained by original author in commit description

> `Fix a bug when calling super's `_compute_product_qty()` was skipping the compute helper from the mixin.`